common/beerocks/bcl/include/bcl/beerocks_logging_custom.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2016-2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+/**
+ * @brief This file helps to define and use a custom logger id which is required for using
+ * easylogging with more than one log files on a single thread.
+ * 
+ * To use custom logger id:
+ * 1. Create beerock::logging object with custom logging id string: "logging_id_str"
+ * 2. Define BEEROCKS_CUSTOM_LOGGER_ID macro as logging_id_str":
+ *    #define BEEROCKS_CUSTOM_LOGGER_ID "logging_id_str"
+ * 3. Include this file where there is a need to write to the non-default logger id.
+ */
```

Comment:
Now that you say that and I look at that, it seems that it will give us the same result.
I didn't know about this option...
I will test it, and it will work, I'll remove this commit, and fix the follow-up commit.
I'm just not sure if it will affect other files on another thread.

Reply:
ELPP_DEFAULT_LOGGER is only used to set kDefaultLoggerId which is a module-local variable. So it doesn't help for functions from libraries. But it can be used for the "main" functions.

Reply:
@arnout 
I tested it with:
```cpp
#define BEEROCKS_CUSTOM_LOGGER_ID BEEROCKS_MONITOR
#include <bcl/beerocks_logging_custom.h>
```
It doesn't work. I see the monitor constructor print on the ap_manager.
I guess it is because the ap_manager is the default logger.
Anyway, I leave it as I wrote it.